src/pages/components/stepper.mdx

```diff
@@ -56,6 +56,27 @@ import IconsCode from '!!raw-loader!../../examples/accordions/stepper/StepperIco
 
 ## API
 
+The Stepper component follows this structure:
+
+```jsx
+<Stepper>
+  <Stepper.Step>
+    <Stepper.Label />
+    <Stepper.Content />
+  </Stepper.Step>
+  {/* other steps */}
+</Stepper>
+```
+
+<!-- markdownlint-disable no-emphasis-as-heading -->
+
+**Stepper orientation**
+
+<!-- markdownlint-enable no-emphasis-as-heading -->
+
+- Content for a vertical stepper must be defined using the `Stepper.Content` for each step
```

Comment:
Another must! You can rearrange it like this: "Define the content for the vertical stepper using the 'Stepper.Content' for each step" The second bullet: "Render the content for a horizontal stepper externally. See example for details."